app/src/main/java/com/ds/avare/storage/Preferences.java

```diff
@@ -361,6 +365,30 @@ public float getAudibleClosingInCriticalAlertRatio() {
         return (float) (Integer.parseInt(mPref.getString(mContext.getString(R.string.AudibleTrafficAlertsClosingCritalAlert), "0")) / 100.00);
     }
 
+    public float getAudibleTrafficAlertsAltitude() {
+        return Float.parseFloat(mPref.getString(mContext.getString(R.string.AudibleTrafficAlertsAltitude), "1000.0"));
```

Comment:
Do all/some of these float conversions require try catch?

Reply:
That is a good point.  Without them the feature just quietly fails (just tested with some alpha chars).  I will put that in now.  Was missing those on the previous version too.